app/models/reservation.rb

```diff
@@ -0,0 +1,9 @@
+class Reservation < ApplicationRecord
+  belongs_to :user
+  belongs_to :car
+
+  validates :reservation_date, presence: true
+  validates :city, presence: true
+  validates :user_id, presence: true
+  validates :car_id, presence: true
```

Comment:
These foreign keys will be only integers, why don't add this for validation?

- Note: I observe on the car model, a lot of `presence: true`validations, but I'm almost sure this could improve by adding validations for integer/string/ and more rules according to car features. I kindly invite you to dig more into this to make a better application. Remember, this is the last project, imagine you're the owner of this page/company. 😄

Reply:
Fixed as advised. Thanks :relaxed: